Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN apt-get update --allow-releaseinfo-change
 
 # System prerequisites
 RUN apt-get update \
- && apt-get -y install build-essential libpq-dev pdftk ghostscript poppler-utils curl \
```

Comment:
I irrelevantly switched from `pdftk` to `pdftk-java` -- we're already using `pdftk-java`; this makes it clearer.

Reply:
remember back in the day when we had an issue with PDF stuff and differences between what was running locally and what was running on CI and production? How can we make sure that what we install locally to our computers matches?

Reply:
`brew bundle install` will install things from `Brewfile` which is what we want.